docs/contributing-building.md

```diff
@@ -69,3 +82,45 @@ there, without needing to supply the `-p` flag:
 cd crates/jit/
 cargo build
 ```
+
+## Cross Compiling Wasmtime
+
+By default `cargo build` will build Wasmtime for the platform you're running the
+build on. You might, however, want to build Wasmtime for a different platform!
+Let's say for example that you want to build Wasmtime for
+`aarch64-unknown-linux-gnu`. First you'll want to acquire the Rust standard
+library for this target:
+
+```shell
+rustup target add aarch64-unknown-linux-gnu
+```
+
+Next you need to install a native C toolchain which has a linker for the desired
+target. This is unfortunately not very easy to acquire on most platforms. It's
+recommended for now to use Linux and use a distribution's package manager to
+acquire this native toolchain. For example Debian-based distributions can
+install the `gcc-aarch64-linux-gnu` package.
```

Comment:
You also need a cross libc; you may want to mention the package name here explicitly.  (The `gcc-aarch64-linux-gnu` on Ubuntu only `Recommends`, but does not require, the corresponding libc packages.)

Reply:
Ah good point, I tried to clarify some wording in that regard.